deepstack_sdk/viz.py

```diff
@@ -51,7 +52,7 @@ def drawResponse(image,response,output_font=cv2.FONT_HERSHEY_SIMPLEX,output_font
 
                 image_arr = cv2.putText(
                     img=image_arr,
-                    text=txt + " ( " + str(100*obj.confidence)+"% )",
+                    text=(txt + (" ( " + str(100*(obj.confidence))+"% )" if show_conf else '' ))if show_label else None,
```

Comment:
- Add a space between `'' ))` and `if` in this line
- I think the value `text` should be populated outside of/before the **putText** function. The current implementation conditions that label must be enabled before confidence can show. On the contrary, people should be able to (1) display only label, (2) display only confidence, (3) display none of the two.

That case, the implementation should look like this.
```suggestion
                    text = ""
                    text += txt if show_label
                    text += " ( " + str(100*obj.confidence)+"% )" if show_conf
                    
                    image_arr = cv2.putText(
                    img=image_arr,
                    text=text,
```